packages/ruleset/test/array-attributes.test.js

```diff
@@ -497,13 +497,11 @@ describe(`Spectral rule: ${ruleId}`, () => {
       };
 
       const results = await testRule(ruleId, rule, testDocument);
-      expect(results).toHaveLength(6);
+      expect(results).toHaveLength(4);
```

Comment:
I had to make adjustments to the "root" testing document to adhere to the API Handbook naming guidance - this resulted in a few of the rules catching less problems since they only happen in the canonical schema, for example